### PR TITLE
Renamed prefix SV to SU (2019R2)

### LINKED ISSUE #15: Rename prefix SV to SU (2019R2)
Update all pages and references to use SU instead of SV in 2019R2 branch.

### PR DESCRIPTION
**Pull request by markoan**
Renamed prefix SV to SU in latest commit for branch 2019R2
- Updated zip
- Renamed all pages and set references to correct name and folder
- Updated GI
- Updated Dashboard
- Updated mobile definition

Closes #15

### CODE REVIEW

File: AcumaticaSurveyPkg/Customization/Pages/SU/SU201000.aspx.cs [hunk 1]
```diff
@@ -7,7 +7,7 @@
 using System.Web.UI.WebControls;
 using System.Web.UI.HtmlControls;
 
-public partial class  Page_SV301000 : PX.Web.UI.PXPage
+public partial class  Page_SU201000 : PX.Web.UI.PXPage
```
Comment (DhirenChhapgar) [inline AcumaticaSurveyPkg/Customization/Pages/SU/SU201000.aspx.cs:10]: This should be SU301000

File: AcumaticaSurveyPkg/Customization/_project/ProjectMetadata.xml [hunk 2]
```diff
@@ -1,2 +1,2 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<project name="AcumaticaSurveyPkg" level="0" description="" />
\ No newline at end of file
+<project name="Customization" level="0" description="" />
```
Comment (DhirenChhapgar) [inline AcumaticaSurveyPkg/Customization/_project/ProjectMetadata.xml:2]: @markoan we need to keep project name as AcumaticaSurveyPkg

File: AcumaticaSurveyPkg/Customization/Pages/SU/SU201000.aspx.cs (hunk 1 above)
Comment (markoan) [reply AcumaticaSurveyPkg/Customization/Pages/SU/SU201000.aspx.cs:10]: THis is weird, I only renamed the SV to SU in the file... Why did it change? and is it correct to have the file named SU201000.aspx.cs refer to the partial class named SU301000?

File: AcumaticaSurveyPkg/Customization/Pages/SU/SU301000.aspx.cs [hunk 3]
```diff
@@ -7,7 +7,7 @@
 using System.Web.UI.WebControls;
 using System.Web.UI.HtmlControls;
 
-public partial class  Page_SV201000 : PX.Web.UI.PXPage
+public partial class  Page_SU301000 : PX.Web.UI.PXPage
```
[the remark this answers is not in the document]
Comment (markoan) [reply AcumaticaSurveyPkg/Customization/Pages/SU/SU301000.aspx.cs:10]: Same here.
I only renamed SV to SU inside the file... Why did it change the number? and is it correct to have the file named SU301000.aspx.cs refer to the partial class named SU201000?

File: AcumaticaSurveyPkg/Customization/_project/ProjectMetadata.xml (hunk 2 above)
Comment (markoan) [reply AcumaticaSurveyPkg/Customization/_project/ProjectMetadata.xml:2]: Got it. It got changed when I exported the content and wanted to keep the same folder (it saves to the folder with the name you use).
Are you renaming this manually each time you export the module?

File: AcumaticaSurveyPkg/Customization/Pages/SU/SU201000.aspx.cs (hunk 1 above)
Comment (markoan) [reply AcumaticaSurveyPkg/Customization/Pages/SU/SU201000.aspx.cs:10]: Double-checked files and contents match so this should not cause issues for next commits.

File: AcumaticaSurveyPkg/Customization/Pages/SU/SU301000.aspx.cs (hunk 3 above)
[the remark this answers is not in the document]
Comment (markoan) [reply AcumaticaSurveyPkg/Customization/Pages/SU/SU301000.aspx.cs:10]: Double-checked files and contents match so this should not cause issues for next commits.

**Pull request merged**